### PR TITLE
create power of infrastructure of code blog

### PR DESCRIPTION
**Pull request by rubyrat**
creating blog post and related images

### CODE REVIEW

File: src/posts/power-of-infrastructure-as-code-iac.md [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+
+Our favourite Infrastructure as Code tool at Verifa is Terraform since it is cloud agnostic and is so widely used that it’s easy to find modules and examples. And the best part is, it’s open source! To show the true power of IaC we’re going to provision some resources in [UpCloud]([https://upcloud.com/](https://upcloud.com/)).
+
+## Creating UpCloud resources through the web console
+
+Creating cloud resources using the cloud provider’s web console is a common starting point when testing a new cloud provider, or just starting out in the cloud in general. This method which consists of mostly clicking mouse buttons is sometimes called ClickOps. Let’s create a virtual machine in UpCloud through the [web console]([https://hub.upcloud.com/](https://hub.upcloud.com/)) by clicking on the “Deploy server” button:
```
Comment (jlarfors) [inline src/posts/power-of-infrastructure-as-code-iac.md:24]: @rubyrat same issue with the link here...

File: src/posts/power-of-infrastructure-as-code-iac.md [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+
+![](/blogs/power-of-infrastructure-as-code-iac/6-upcloud-select-resources.png)
+
+After a summary step we get the Terraform configuration and also handy commands on how to import this resource into Terraform [state]([https://www.terraform.io/language/state](https://www.terraform.io/language/state)) which means we can start managing the resource using Terraform now:
+
+![](/blogs/power-of-infrastructure-as-code-iac/7-upcloud-export-to-terraform.png)
+
+Let’s follow this process to import the resources into Terraform and see if there’s any benefit of doing so. Here’s the complete configuration for the virtual machine:
+
+```jsx
```
Comment (jlarfors) [inline src/posts/power-of-infrastructure-as-code-iac.md:58]: @rubyrat the language should be `hcl` not `jsx`...

File: src/posts/power-of-infrastructure-as-code-iac.md (hunk 1 above)
Comment (rubyrat) [reply src/posts/power-of-infrastructure-as-code-iac.md]: done!

File: src/posts/power-of-infrastructure-as-code-iac.md (hunk 2 above)
Comment (rubyrat) [reply src/posts/power-of-infrastructure-as-code-iac.md]: done!

**Comment by rubyrat** [comment]
updated the text for the kicker and the summary now. All looks good, so I'll merge to main.

**Pull request merged**